app/models/game.rb

```diff
@@ -1,8 +1,19 @@
 class Game < ActiveRecord::Base
-
+  belongs_to :user
+  COMPLEXITIES = {
+    1 => "low",
+    2 => "medium",
+    3 => "high",
+    4 => "extreme"
+  }
   validates :name, presence: true, uniqueness: true
```

Comment:
We thought there should be some sort of differentiation between games with the same title. Game-specific expansions can be explicitly stated in the name.